events/event/views.py

```diff
@@ -34,6 +34,31 @@ def get(self, request, pk=None):
         else:
             return HttpResponseForbidden('Permission denied')
 
+    def post(self, request):
+        try:
+            event_data = json.loads(request.body.decode())
+        except Exception as e:
+            return JsonResponse({
+                "json error": e.message,
+            }, status=400)
+        validation_form = EventCreateForm(event_data)
+        if validation_form.is_valid():
+            event = Event()
+            event.title = event_data.get('title')
+            event.start_date = event_data.get('start_date')
+            event.end_date = event_data.get('end_date')
+            event.location = event_data.get('location')
+            event.description = event_data.get('description')
+            event.owner_id = event_data.get('owner_id')
```

Comment:
@sbufan. No need to expect this data from front-end. In this case you could use:
`event.owner_id = request.user.id`